spk/ffmpeg/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = ffmpeg
 SPK_VERS = 4.3.2
-SPK_REV = 37
+SPK_REV = 38
```

Comment:
Kind reminder to test/build with `CMAKE_USE_NINJA=0` to check the cmake changes.
FYI you can now build without such a commit. In your fork, use the `Run Workflow` button: https://github.com/SynoCommunity/spksrc/wiki/Publish-a-package-from-Github#run-buildpublish-workflow

Reply:
This is really really nice.  Great documentation btw, thnx.

Reply:
I made some mileage from your advice, I've set `CMAKE_USE_NINJA=0` to default and only forced `libaom` to build using `ninja`.  github action build outputs looks pretty good besides `ffsync` and `squidguard` (considering `minio` on dsm7 is being looked at in #4683)

Did any of the two ever built recently?

Reply:
Actually, an option to set something like `.BROKEN` empty file in the known broken spk could allow github action to NOT build them, using less CPU cycles and providing better outputs.

Reply:
@publicarray problem now solved.  Using `COMPILE_MAKE_OPTIONS` it was trivial to remove almost all `COMPILE_TARGET` calls making almost every dependencies to now "parallel" build.